src/lib/teads-curve/README.md

```diff
@@ -1,55 +1,54 @@
-# Teads' CPU Estimation Model
+# Teads' CPU Estimation Plugin
 
-> [!NOTE] > `Teads-Curve` is a community model, not part of the IF standard library. This means the IF core team are not closely monitoring these models to keep them up to date. You should do your own research before implementing them!
+> [!NOTE] > `Teads-Curve` is a community plugin, not part of the IF standard library. This means the IF core team are not closely monitoring these plugins to keep them up to date. You should do your own research before implementing them!
 
-Teads Engineering team has built a model that is capable of estimating CPU usages across varying type of CPUs using a curve commonly known as Teads Curve.
+Teads Engineering team has built a plugin that is capable of estimating CPU usages across varying type of CPUs using a curve commonly known as Teads Curve.
 
-## Model name
+## Plugin name
 
-IF recognizes the Teads CPU model as `teads-curve`.
+IF recognizes the Teads CPU plugin as `teads-curve`.
 
 ## Parameters
 
-### Model config
+### Plugin config
```

Comment:
@jmcook1186 we should discuss how to make this clear. It's used as defaults in example, but title is config.